tooling/charts/tl500-course-content/templates/crw/ca-configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.crw.gitCustomCA }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "20"
+  name: che-git-self-signed-cert
+  namespace: tl500-workspaces
```

Comment:
Should probably use `{{ .Values.crw.namespace | default "tl500-workspaces" | quote }}` here